dev/scripts/main.js

```diff
@@ -249,29 +249,29 @@ bikeApp.getCityBikesPricing = function(userTime, distanceDurationMinutes, roundT
 	.then(function(res) {
 		//24 hour pricing divided by 24 to obtain hourly rate
 		const hourlyPrice = (res.data.plans[1].price) / 24;
+		//multiply user hours to hourly price
 		const userHourlyPrice = (userTime * hourlyPrice).toFixed(2);
+		console.log(userHourlyPrice, "should be 0.66")
 		bikeApp.displayResults(distanceDurationMinutes, roundTripTime, userHourlyPrice, hourlyPrice); 
 	});
 };	
 
 
-bikeApp.displayResults = function (stationDistance, roundTripTravelTime, userRoundTripPrice, userPrice) {
+bikeApp.displayResults = function (stationDistance, roundTripTravelTime, userFinalPrice, pricePerHour) {
```

Comment:
ok can you just remove the pricePerHour parameter here since it is not being used